src/software/geom/util.h

```diff
@@ -601,8 +570,20 @@ std::optional<Segment> mergeOverlappingParallelSegments(Segment segment1,
  *         Returns std::nullopt if the segments aren't parallel, arem't overlapping, or
  * aren't redundant
  */
```

Comment:
Looks like we need to update this jdoc?

Reply:
I moved some stuff around and I don't see what you mean.

What's wrong with the javadoc?

Reply:
Oh, the indenting is broken, but I realized that it's not part of your changes. Out-of-scope, so your call if you want to fix it or not.

Reply:
I'm going to leave it out of this PR